addons/hr_presence/models/hr_employee.py

```diff
@@ -117,28 +104,29 @@ def write(self, vals):
         return super().write(vals)
 
     def action_open_leave_request(self):
-        self.ensure_one()
-        return {
-            "type": "ir.actions.act_window",
-            "res_model": "hr.leave",
-            "views": [[False, "form"]],
-            "view_mode": 'form',
-            "context": {'default_employee_id': self.id},
-        }
+        leave_type = self.env['hr.leave.type'].search([('requires_allocation', '=', 'no')], limit=1)
+        vals = [{
+            'employee_id': employee.id,
+            'name': _("Request manually created with the Presence Control wizard"),
+            'holiday_status_id': leave_type.id,
+            'state': 'confirm',
+        } for employee in self]
+        self.env['hr.leave'].create(vals).action_validate()
+        self._action_set_manual_presence(False)
+
+        for employee in self:
+            employee.message_post(body=_("A sick leave has been created to cover employee's absence."))
```

Comment:
On a real database, you'll probably never set a sick time off, but you'll take a random one.

Reply:
Example on runbot. Set Sick Time Off as "Need an allocation", you'll end up with an unpaid.